.travis.yml

```diff
@@ -60,10 +60,6 @@ install:
       fi
     fi
 
-  # Perceptual image hashing (TBD: push recipe to conda-forge!)
```

Comment:
It may be worth noting that the reason imagehash was put in conda-recipe-scitools rather than conda-forge is that conda-forge doesn't currently work our travis testing.